This patch allows the module nativescript-nodeidy to work on
NativeScript v>4.1. It is taken from https://github.com/EddyVerbruggen/nativescript-nodeify/pull/50
--- node_modules/nativescript-nodeify/patch-npm-packages.js
+++ node_modules/nativescript-nodeify/patch-npm-packages.js
@@ -1,7 +1,7 @@
 module.exports = function ($logger, $projectData, $usbLiveSyncService) {
   var liveSync = $usbLiveSyncService.isInitialized;

-  if (liveSync) {
+  if (!liveSync) {
     return;
   }
